### PR TITLE
Add reviews to paid providers

### LINKED ISSUE #131: A Suspected Issue with 1 of your Paid IPTV Recommendations
Under the heading Providers/Paid you list BestBuyIPTV but be careful.
When I was looking for an IPTV source I came across these but looking on TrustPilot:-
https://www.trustpilot.com/review/bestbuyiptv.com
I became very suspicious and my suspicions were later confirmed with TroyPoint:-
https://troypoint.com/bestbuyiptv/
Also, they have multiple sites:-
https://bestbuyiptv.us/en2/?utm_medium=company_profile&utm_source=trustpilot&utm_campaign=domain_click
and always, the review is to treat with caution at best and more commonly, stay away.

If you want a recommendation you could add:-
https://bestiptvuk.me
I only had a day or two to sort. an IPTV provider before the British & Irish Lions rugby tour of South Africa was due to start.
I struck gold with these people. I have been with them for 10 months. They are not perfect but for the price I can truthfully say that they are damn good. Their free trial is farcical as it is 6 hours with no sports or only 2 hours with sports.
Also, and most important for me, they offer upto 7 days of catch-up on many UK and International streams.
cheersD
p.s. This is my 1st comment on Github so if you reply I may not know how to read you mail.

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Addresses issue #131 based on Trustpilot review ratings + A message asking to deal with caution if no review URL concerning the website has been found

Close #131

### CODE REVIEW

**Comment by freearhey** [comment]
Can we just add a link to the reviews next to it?

```
- [BestBuyIPTV](https://bestbuyiptv.com/) - Over 7300 HD channels and 9600 VOD 1080p, of more than 38 countries. ([reviews](https://www.trustpilot.com/review/bestbuyiptv.com))
```

**Comment by LaneSh4d0w** [comment]
Well, it depends on our point of view I suppose. If we promote poorly rated providers, it may have a bad image on us.

**Comment by ront735** [comment]
Really the best

**Comment by bbbureneecartoonz** [comment]
merge pls pls pls and review this all 2 people pls

**Comment by aDNpz** [comment]
i'm currently working on their resellpanel (https://reselleriptv.org/). it seems that they are actually the reseller provider of all. pls correct me if i'm wrong.
even xtream-ui.org is made by them. i'm currently on their reselling panel and everything seems to work fine. the payment process made no problems (paid by crypto).

**Comment by LaneSh4d0w** [comment]
@freearhey what do you think we should do for this PR ?

**Comment by freearhey** [comment]
@LaneSh4d0w Well, my opinion hasn't changed. If anyone has any suspicions about any of the sites, I think it would be better if we would just add links to them in the description, so that people can explore everything themselves and decide whether to use such a service or not. The same rule of course should apply to positive feedback as well.

**Comment by LaneSh4d0w** [comment]
I'll make the necessary changes then.

**Comment by LaneSh4d0w** [comment]
LMK if I need to make more changes.

File: README.md [hunk 1]
```diff
@@ -97,10 +97,11 @@ List of IPTV providers.
...
-- [BestBuyIPTV](https://bestbuyiptv.com/) - Over 7300 HD channels and 9600 VOD 1080p, of more than 38 countries.
-- [Happy Watch 99](https://happywatch99.com/) - Live Khmer & Thai TV, Movies and Series.
-- [Necro IPTV](https://necroiptv.com) - Offers all premium UK, Ireland, Germany, Turkey, Arabic, USA and Canada Channels.
+- [IPTV Shop](https://iptv.shop/) - Over 6000 live TV channels + 4000 Movies & TV Shows (VOD). (Trustpilot has disabled their review page, proceed with caution !)
+- [Edem.tv](https://edem.tv/) - Over 400 Russian IPTV channels and archive of programs for 4 days. [(reviews)](https://trustpilot.com/review/edem.tv)
+- [BestBuyIPTV](https://bestbuyiptv.com/) - Over 7300 HD channels and 9600 VOD 1080p, of more than 38 countries. [(reviews)](https://trustpilot.com/review/bestbuyiptv.com)
+- [Happy Watch 99](https://happywatch99.com/) - Live Khmer & Thai TV, Movies and Series. (No reviews available, proceed with caution !)
+- [Necro IPTV](https://necroiptv.com) - Offers all premium UK, Ireland, Germany, Turkey, Arabic, USA and Canada Channels. [(reviews)](https://trustpilot.com/review/necroiptv.com)
```
Comment (freearhey) [inline README.md:104]:
```suggestion
- [IPTV Shop](https://iptv.shop/) - Over 6000 live TV channels + 4000 Movies & TV Shows (VOD).
- [BestBuyIPTV](https://bestbuyiptv.com/) - Over 7300 HD channels and 9600 VOD 1080p, of more than 38 countries. ([reviews](https://trustpilot.com/review/bestbuyiptv.com))
- [Happy Watch 99](https://happywatch99.com/) - Live Khmer & Thai TV, Movies and Series.
- [Necro IPTV](https://necroiptv.com) - Offers all premium UK, Ireland, Germany, Turkey, Arabic, USA and Canada Channels. ([reviews](https://trustpilot.com/review/necroiptv.com))
```

I think we can dispense with the warnings, and when there are reviews just add a link to them.

**Review by freearhey — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**